README.md

```diff
@@ -22,4 +22,26 @@ The aggregator will retrieve log entries from all known servers and store the re
 3. **Update all logs beginning at date**
   ```
   python aggregator/aggregator.py --date 2015-03-01
-  ```
\ No newline at end of file
+  ```
+
+
+## Simulation 
+
+### Description
+
+### Before you run the simulation
+
+1. **Install pyipinfodb (an API for transfering ip address to geolocation) and geopy** 
+  ```
+  pip install git+git://github.com/markmossberg/pyipinfodb.git
+  
+  pip install geopy
```

Comment:
Can you add these two required libraries in the `requirements.txt` file, so that the deployment script can install it on the fly?